armadillo/src/test/java/org/molgenis/armadillo/controller/StorageControllerTest.java

```diff
@@ -89,6 +90,108 @@ void uploadObject() throws Exception {
             mockSuAuditMap(Map.of(PROJECT, "lifecycle", OBJECT, "core/nonrep2.parquet"))));
   }
 
+  @Test
+  void uploadCsvObject() throws Exception {
```

Comment:
rename to testUploadCsvObject ? or don't we prefix all with 'test' in Armadillo?